boxoffice/views/admin_discount.py

```diff
@@ -52,12 +62,38 @@ def jsonify_discount_policies(data_dict):
     )
 @render_with({'text/html': 'index.html', 'application/json': jsonify_discount_policies}, json=True)
 def admin_discount_policies(organization):
+    RESULTS_PER_PAGE = 6
     query = request.args.get('search')
```

Comment:
I suggest renaming this to `search_query`. It's easy to confuse this with SQLAlchemy's `query.`